main.py

```diff
@@ -115,4 +115,6 @@ def playsong(event):
 if __name__ == "__main__":
     root = ctk.CTk()
     app = musicplayer(root)
+    root.wm_iconbitmap(default="icon.ico")
```

Comment:
This icon is not in the repo? I've never seen it in the git history?

Reply:
Added icons, turns out ICO is windows specific

PNG should be the better alternative